pkgs/servers/sql/postgresql/ext/citus.nix

```
@@ -0,0 +1,36 @@
+{ lib, stdenv, fetchFromGitHub, postgresql, curl, lz4 }:
+
+stdenv.mkDerivation rec {
+  pname = "citus";
+  version = "11.3.0";
+
+  src = fetchFromGitHub {
+    owner  = "citusdata";
+    repo   = "citus";
+    rev    = "v${version}";
+    sha256 = "e+pkKFFFIATIoODRmOGFr0yFCYo2Q7mVJJ/zbtObJ30=";
+  };
+
+  buildInputs = [ postgresql curl lz4 ];
+  makeFlags = [ "PREFIX=$(out)" ];
+
+  preInstall = ''
+    mkdir -p $out
+  '';
+
+  installPhase = ''
+    runHook preInstall
+  '';
```

I think this can be refactored as such. But I'm a bit confused, as it doesn't seem to be doing anything.

```suggestion
  installPhase = ''
    runHook preInstall
    mkdir -p $out
    runHook postInstall
  '';
```